packages/block-library/src/navigation/index.php

```diff
@@ -257,10 +257,26 @@ function block_core_navigation_get_classic_menu_fallback() {
 	$classic_nav_menus = wp_get_nav_menus();
 
 	// If menus exist.
-	if ( $classic_nav_menus && ! is_wp_error( $classic_nav_menus ) && count( $classic_nav_menus ) === 1 ) {
-		// Use the first classic menu only. Handles simple use case where user has a single
-		// classic menu and switches to a block theme. In future this maybe expanded to
-		// determine the most appropriate classic menu to be used based on location.
+	if ( $classic_nav_menus && ! is_wp_error( $classic_nav_menus ) ) {
+		// Handles simple use case where user has a classic menu and switches to a block theme.
+
+		// Returns the menu assigned to location `primary`.
+		$locations = get_nav_menu_locations();
+		if ( isset( $locations['primary'] ) ) {
+			$primary_menu = wp_get_nav_menu_object( $locations['primary'] );
```

Comment:
Again I'm not sure if this string can be in different cases. If so should be normalize to lowercase and then run the comparison?

Reply:
This is the case of returning a menu which is currently used in the location named primary, not menu with the exact name of primary — so I don't think we need to do any normalization?

Reply:
Ah so this case is ok. Gotcha 👍